websites/C/Coup Critique/metadata.json

```
@@ -0,0 +1,50 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "240521747852558347",
+		"name": "RisingSunLight"
+	},
+	"service": "Coup Critique",
+	"description": {
+		"en": "Tutorial list about Pokemon strategy and french competitive scene. You will find helpful vocabulary articles and practices in the Pokemon strategy.",
+		"fr": "Liste des guides sur la stratégie Pokémon et la scène compétitive française. Vous y retrouverez des articles d'aide sur le vocabulaire et les pratiques de la stratégie Pokémon."
+	},
+	"url": "www.coupcritique.fr",
```

Site supports non-www domain.
```suggestion
	"url": [
		"www.coupcritique.fr",
		"coupcritique.fr"
	],
```